Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -2510,10 +2512,7 @@ RefPtr<ByteArrayPixelBuffer> CanvasRenderingContext2DBase::cacheImageDataIfPossi
 RefPtr<ImageData> CanvasRenderingContext2DBase::makeImageDataIfContentsCached(const IntRect& sourceRect, PredefinedColorSpace colorSpace) const
 {
     if (std::holds_alternative<CachedContentsTransparent>(m_cachedContents)) {
-        auto imageData = ImageData::create(sourceRect.size(), colorSpace);
-        if (imageData)
-            imageData->data().zeroFill();
-        return imageData;
+        return ImageData::createZeroFilled(sourceRect.size(), colorSpace);
```

Comment:
After further reflection, I've decided to keep m, because I think it's a safer API to have:
one function `createZeroFilled()` that returns a fully-initialized object;
rather than a combination of `create()` returning an unfinished ImageData object that should not be used and then a `data()->zeroFill` that magically does the right thing to finalize the object.
Also, it makes for a nice one-liner `return`.

But please let me know if you disagree, and you think I really should keep the Create+zeroFill combination (and could you please explain why you'd prefer that?).